### DISCUSSION TITLE
FFT-Water

### DISCUSSION THREAD

**Discussion by r4shpro** [Try/OpenGothic#427] [Ideas]
Greetings guys!

Would it be possible to implement FFT-based water?

There was a DX11 implementation attempt some time ago. The [Water](https://www.youtube.com/watch?v=QN5_0SrHF58) does look stunning, shaders and Furrier-based waves.

**Comment by Try** [comment]
FFT water doesn't look promising at all for 'ground-based' game, since it has passive cost. Basically we will lose fps on frames that do not have any water that way.

Deferred water, with Gerstner waves work way better:
[![IMAGE ALT TEXT](http://img.youtube.com/vi/-k4uZYFOoGU/0.jpg)](http://www.youtube.com/watch?v=-k4uZYFOoGU "Video Title")
